packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -421,6 +423,18 @@ export class Connector implements IConnector {
                 throw e;
             }
         }
+    }
+
+    private async unlock() {
+        if (this._isLocked) {
+            await unlock(this._lockFile);
```

Comment:
I had to stare at this a bit before I realized it wasn't a recursive call. Changing the names of the private `lock` and `unlock` methods would make this clearer.

Reply:
changed them to `(un)lockFile` because I'm not very original today. I gladly accept better suggestions (and even worse ones if they are funny 😛)

Reply:
Good enough for me. 👍